lib/tests/dynamicRow.validation.test.js

```diff
@@ -0,0 +1,85 @@
+import { extractSubElementId } from "../validation";
```

Comment:
Is there  any particular reason a separate test file exists. There is already a validation test file

Reply:
Dynamic row will get more complicated as time goes so i though it might be good idea to separate. But yes it can be merged into it.